test/powershell/Language/Classes/scripting.enums.tests.ps1

```diff
@@ -87,4 +121,5 @@ Describe 'Basic enum errors' -Tags "CI" {
     ShouldBeParseError 'enum foo { e = $foo }' EnumeratorValueMustBeConstant 15 -SkipAndCheckRuntimeError
     ShouldBeParseError 'enum foo { e = "hello" }' CannotConvertValue 15 -SkipAndCheckRuntimeError
     ShouldBeParseError 'enum foo { a;b;c;' MissingEndCurlyBrace 10
+    ShouldBeParseError 'enum foo : string { a }' InvalidUnderlyingType 11 -SkipAndCheckRuntimeError
```

Comment:
Can we add cases for value to small and too large?

Reply:
I'm fine with one check for the error case.

Reply:
@TravisEz13 Do we want to update `ParserStrings.EnumeratorValueTooLarge` is "either too large _or_ too small for a [typename]". 

Currently (since 5.0) we throw:

```powershell
PS C:\>enum E { A = [long]::MinValue }
At line:1 char:13
+ enum E { A = [long]::MinValue }
+              ~~~~~~~~~~~~~~~~
Enumerator value is too large for a System.Int.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : EnumeratorValueTooLarge
```

Reply:
Sure